lib/YuiRestClient/Widget/Tab.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YuiRestClient::Widget::Tab;
```

Comment:
Could you please put this change into separate commit?

Reply:
Will do. Here I could use menucollection instead of creating new Widget, but I thought the naming is confusing for someone who isn't very familiar, WDYT?

Reply:
I prefer tab than dumbtab :) as we did in https://github.com/qe-yast/ruby-yui-rest-client/blob/master/lib/yui_rest_client/widgets/tab.rb